### ISSUE TITLE
How to change default table name?

### ISSUE THREAD

**Issue by marbuser** [mweibel/connect-session-sequelize#85]
So by default, a table called `Sessions` gets created and we can define a custom table but it needs to ALREADY exist which is sort of annoying.

Is it possible we could get some sort of config option to just state we want to rename the `Sessions` table, but we still want it to be created for us?

This would be useful for those of us that need more than 1 session table.

**Comment by nullivex** [comment]
Like I did here

```js
        store: new SequelizeStore({
          db: K.db.sequelize,
          table: 'StaffSession'
        }),
```

**Comment by marbuser** [comment]
> Like I did here
> 
> ```js
>         store: new SequelizeStore({
>           db: K.db.sequelize,
>           table: 'StaffSession'
>         }),
> ```
> 
> I have used this on a lot of new databases it does create the proper table name. Is there something else I can answer?

@nullivex Yeah I tried that. When I do that I get a nasty error like so;
```
/node_modules/connect-session-sequelize/lib/connect-session-sequelize.js:58
    return this.sessionModel.sync()
                             ^

TypeError: Cannot read property 'sync' of undefined
    at SequelizeStore.sync (/node_modules/connect-session-sequelize/lib/connect-session-sequelize.js:58:30)
    at Object.sync (/src/middleware/session.js:25:7)
    at Module._compile (internal/modules/cjs/loader.js:721:30)
    at Module._compile (/node_modules/pirates/lib/index.js:83:24)
    at Module._extensions..js (internal/modules/cjs/loader.js:732:10)
    at Object.newLoader [as .js] (/node_modules/pirates/lib/index.js:88:7)
    at Module.load (internal/modules/cjs/loader.js:620:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:560:12)
    at Function.Module._load (internal/modules/cjs/loader.js:552:3)
    at Module.require (internal/modules/cjs/loader.js:657:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/src/routes/session.js:5:1)
    at Module._compile (internal/modules/cjs/loader.js:721:30)
    at Module._compile (/node_modules/pirates/lib/index.js:83:24)
    at Module._extensions..js (internal/modules/cjs/loader.js:732:10)
    at Object.newLoader [as .js] (/node_modules/pirates/lib/index.js:88:7)
    at Module.load (internal/modules/cjs/loader.js:620:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:560:12)
    at Function.Module._load (internal/modules/cjs/loader.js:552:3)
    at Module.require (internal/modules/cjs/loader.js:657:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/src/startup/routes.js:7:1)
```

**Comment by nullivex** [comment]
@marbuser You are right, I didnt see the other part of my definition. I define the `StaffSession` model and then tell it the name, which explains your issue.

For me it was useful being able to define the model, I can see that being a hassle though. @mweibel  any thoughts?

```js
/**
 * Exporting the model
 * @param {object} sequelize
 * @param {object} DataTypes
 * @return {object}
 */
module.exports = function(sequelize,DataTypes) {
  return sequelize.define('StaffSession',{
      sid: {
        type: DataTypes.STRING,
        primaryKey: true
      },
      expires: {
        type: DataTypes.DATE
      },
      data: {
        type: DataTypes.TEXT
      }
    },
    {
      indexes: [
        {
          name: 'expires_index',
          method: 'BTREE',
          fields: ['expires']
        },
        {
          name: 'createdAt_index',
          method: 'BTREE',
          fields: ['createdAt']
        },
        {
          name: 'updatedAt_index',
          method: 'BTREE',
          fields: ['updatedAt']
        }
      ]
    })
}
```

**Comment by marbuser** [comment]
> @marbuser You are right, I didnt see the other part of my definition. I define the `StaffSession` model and then tell it the name, which explains your issue.
> 
> For me it was useful being able to define the model, I can see that being a hassle though. @mweibel any thoughts?
> 
> ```js
> /**
>  * Exporting the model
>  * @param {object} sequelize
>  * @param {object} DataTypes
>  * @return {object}
>  */
> module.exports = function(sequelize,DataTypes) {
>   return sequelize.define('StaffSession',{
>       sid: {
>         type: DataTypes.STRING,
>         primaryKey: true
>       },
>       expires: {
>         type: DataTypes.DATE
>       },
>       data: {
>         type: DataTypes.TEXT
>       }
>     },
>     {
>       indexes: [
>         {
>           name: 'expires_index',
>           method: 'BTREE',
>           fields: ['expires']
>         },
>         {
>           name: 'createdAt_index',
>           method: 'BTREE',
>           fields: ['createdAt']
>         },
>         {
>           name: 'updatedAt_index',
>           method: 'BTREE',
>           fields: ['updatedAt']
>         }
>       ]
>     })
> }
> ```

Yep this managed to work for me. I think in terms of this actual package though it would be a hell of a lot easier to make the model name customisable for those that still want the default table provided, but just need to rename it.

**Comment by mweibel** [comment]
I guess that shouldn't be a hassle to make a PR for it and I'd be open for it. 
There's another PR (#84) currently open which refactors quite some stuff and I plan to merge it in very soon. Just so you know that there might be merge conflicts.

**Comment by igordeoliveirasa** [comment]
How is it going about the table renaming feature? Thanks

**Comment by kennethacohen** [comment]
Running into this issue as well - is then only resolution at this point to just make a custom Session model?

**Comment by mweibel** [comment]
as said, I'd accept a PR which allows to change the table name.

**Comment by durek1337** [comment]
I'll try it. It should be only a few minutes of work.

**Comment by mweibel** [comment]
Released version 6.1.0 with this fix.
